config/settings/base.py

```diff
@@ -267,4 +267,10 @@
 SALESFORCE_LOGIN_URL = "https://login.salesforce.com"
 
 # Amazon
-AWS_S3_DRAFT_DIR = "draft/"
+AWS_S3_DRAFT_IMG_DIR = 'images/draft/'
+AWS_S3_PUBLIC_IMG_DIR = 'images/public/'
+AWS_S3_DRAFT_HTML_REPOSITORY_DIR = 'html/draft/'
+AWS_S3_PUBLISHED_HTML_REPOSITORY_DIR = 'html/public/'
+
+# Local directories
+LOCAL_REPOSITORY_CACHE = '/tmp/sfdoc_html_repository_cache/'  # local transient file cache for speeding up downloads
```

Comment:
Okay, Heroku seems to return /tmp and I can override it with an environment variable on my own computer. I was slightly hesitant because I don't want it to end up in 

'/var/folders/q4/9bjz2lm10nsfv2z3l_pqjtt40000gq/T'

(the default on my computer today)

It just makes debugging harder. But I can use the TMPDIR environment variable to override it and Heroku uses /tmp.

Reply:
62f989989988580d06342b2279d9de58b2c0d6b7